packages/mobile/src/web3/saga.ts

```diff
@@ -350,6 +351,19 @@ export function* getConnectedUnlockedAccount() {
   const account: string = yield call(getConnectedAccount)
   const result: UnlockResult = yield call(unlockAccount, account)
   if (result === UnlockResult.SUCCESS) {
+    const signedMessage = yield select(signedMessageSelector)
```

Comment:
Is this called when an account is created or imported? Otherwise I think that's a good place to store the signed message as well :)

Reply:
oops 🙈 it's there now

Reply:
the annoying thing is, both the create and restore flows call the `initializeAccount` saga BUT the firebase saga where the fcmToken needs to be updated is kicked off by the root saga so there is a race condition to get the account created/get the signed message stored beforehand. for the create flow, this seems to work okay. but for the restore flow, `initializeAccount` is called too late. so i've also put the generateSignedMessage also in the import saga, in case this looks strange